CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 1.25.0 -dev
+* Feature - Add Nvidia GPU support for p2 and p3 instances
+* Feature - Add support for Elastic Inference
```

Comment:
EI is not yet available in public, so i don't think you need to add this

Reply:
okay